ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitter.scala

```diff
@@ -281,7 +281,7 @@ private[kvutils] class TransactionCommitter(
       .fold((true, keys)) {
         case (
             (allUnique, existingKeys),
-            (_, exe @ Node.NodeExercises(_, _, _, _, _, _, _, _, _, _, _, _, _)))
+            (_, exe @ Node.NodeExercises(_, _, _, _, _, _, _, _, _, _, _, _, _, _)))
```

Comment:
This looks horrible. Can't we do something like this?
```suggestion
            (_, exe: Node.NodeExercises))
```
We have the same in many other places too. We should clean this up a bit in a separate PR. I'm happy to do it myself.

Reply:
In this case you have to write something like: 

```(_, exe: Node.NodeExercises[_, ContractId, VersionnedValue[ContractId]))```
or using type alias 
```(_, exe: Node.NodeExercises.WithTxValue[_, ContractId]))```

I really dislike matching with parametric types.  
But if you whish I will do the "cleanup".

Reply:
Can we do the following?
```scala
(_, exe: Node.Exercises[_, _, _])
```
I mean, we don't really care about the actual type arguments, do we?

Reply:
I suspect in some place this broken the type inference. 
In this case, it seems to compile. However, this definitively broke the type inference of Intellij :-(

Reply:
That's weird that it breaks (IntelliJ's) type inference. The pattern `Node.NodeExercises(_, _, _, _, _, _, _, _, _, _, _, _, _, _)` doesn't make any statements about types whatsoever. Anyway, let's not touch this then for now.